curriculum/js2/4.js

```diff
@@ -4,8 +4,11 @@
  * @returns {number}
  */
 
-const solution = (a) => {
-  return 0;
+const solution = (a, idx = 0, sum = 0) => {
+  if (idx === a.length) {
+    return a.length == 0 ? 0 : sum;
```

Comment:
your base case is already good. once your index reaches the length of the array, you can return something. So why do you need to add another ternary operator? why not just return sum?